### PR TITLE
fix(status-page): gas price should use l2provider, add descriptions and tooltips

### PR DESCRIPTION
**Pull request by cyberhorsey**

### CODE REVIEW

**Comment by RogerLamTd** [comment]
Not sure if this is intentional or coincidental, it looks like gas price is truncated to 1 decimal at the moment?

**Comment by cyberhorsey** [comment]
> Not sure if this is intentional or coincidental, it looks like gas price is truncated to 1 decimal at the moment?

I dont think it is truncated, I think thats the current value

**Review by RogerLamTd — DISMISSED** [review]

File: packages/status-page/src/pages/home/Home.svelte [hunk 1]
```diff
@@ -133,6 +146,8 @@
       colorFunc: (value: Status) => {
         return "green";
       },
+      tooltip:
+        "The ID that the next proposed block on the TaikoL1 smart contract will receive",
```
Comment (d1onys1us) [inline packages/status-page/src/pages/home/Home.svelte:150]:
```suggestion
        "The ID that the next proposed block on the TaikoL1 smart contract will receive.",
```

File: packages/status-page/src/pages/home/Home.svelte [hunk 2]
```diff
@@ -122,6 +133,8 @@
       colorFunc: (value: Status) => {
         return "green";
       },
+      tooltip:
+        "The most recently verified Layer 2 block on the TaikoL1 smart contract",
```
Comment (d1onys1us) [inline packages/status-page/src/pages/home/Home.svelte:137]:
```suggestion
        "The most recently verified Layer 2 block on the TaikoL1 smart contract.",
```

File: packages/status-page/src/pages/home/Home.svelte [hunk 3]
```diff
@@ -99,6 +107,7 @@
         if (value.toString() === "true") return "red";
         return "green";
       },
+      tooltip: "Whether the Taiko smart contract on Layer 1 has been halted",
```
Comment (d1onys1us) [inline packages/status-page/src/pages/home/Home.svelte:110]:
```suggestion
      tooltip: "Boolean describing whether the Taiko smart contract on Layer 1 has been halted.",
```

Comment (wolfderechter) [reply packages/status-page/src/pages/home/Home.svelte:110]: I don't think using 'boolean' is a good idea here

**Review by dionysuzx — DISMISSED** [review]

File: packages/status-page/src/pages/home/Home.svelte (hunk 3 above)
Comment (d1onys1us) [reply packages/status-page/src/pages/home/Home.svelte:110]: I do missing sometimes tho

File: packages/status-page/src/pages/home/Home.svelte [hunk 4]
```diff
@@ -150,17 +165,20 @@
...
-      provider: l1Provider,
-      contractAddress: l1TaikoAddress,
+      provider: l2Provider,
+      contractAddress: "",
       header: "Gas Price (gwei)",
       intervalInMs: 20000,
       colorFunc: (value: Status) => {
         return "green";
       },
+      tooltip: "The current recommended gas price for a transaction on Layer 2",
```
Comment (RogerLamTd) [inline packages/status-page/src/pages/home/Home.svelte:181]:
```suggestion
      tooltip: "The current recommended gas price for a transaction on Layer 2.",
```
for consistency

File: packages/status-page/src/pages/home/Home.svelte (hunk 3 above)
Comment (RogerLamTd) [inline packages/status-page/src/pages/home/Home.svelte:110]:
```suggestion
      tooltip: "Whether the Taiko smart contract on Layer 1 has been halted.",
```
for consistency

**Review by RogerLamTd — APPROVED** [review]

**Review by mfinestone — APPROVED** [review]

**Pull request merged**